controllers/storageclaim_controller.go

```diff
@@ -398,15 +404,33 @@ func (r *StorageClaimReconciler) reconcilePhases() (reconcile.Result, error) {
 				// storageclaim is clusterscoped resources using its
 				// hash as the clusterID
 				data["clusterID"] = r.storageClaimHash
+				labels["ramendr.openshift.io/storageID"] = storageID
 				if resource.Name == "cephfs" {
-					volumeSnapshotClass = r.getCephFSVolumeSnapshotClass(data)
+					volumeSnapshotClass = r.getCephFSVolumeSnapshotClass(data, labels)
```

Comment:
data is specific and labels are generic, no need to club them in here as I believe we already have a util which adds labels to objects, pls use that.

Reply:
Data and Labels are assigned to completely different fields in the CR, therefore they must be sent over as separate arguments.

Reply:
Labels always go into object meta on any resource.

Reply:
And? This doesn't contradict what I wrote.